includes/class-image-optimisation.php

```
@@ -162,118 +162,118 @@ public function preload_images() {
 		 * @return string Modified HTML content buffer.
 		 */
 		public function maybe_serve_next_gen_images( $buffer ) {
-			if ( isset( $this->options['image_optimisation']['convertImg'] ) && (bool) $this->options['image_optimisation']['convertImg'] ) {
-				$conversion_format = $this->options['image_optimisation']['conversionFormat'] ?? 'webp';
+			if ( empty( $this->options['image_optimisation']['convertImg'] ) ) {
+				return $buffer;
+			}
 
-				$exclude_imgs = $this->exclude_convert_imgs;
+			$conversion_format = $this->options['image_optimisation']['conversionFormat'] ?? 'webp';
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$conversion_format'. (undefined)

(UnusedLocalVariable)